### PR TITLE
refactor: reintroduce the .NET Framework 4.8 target framework

### PR DESCRIPTION
**Pull request by Dor-bl**
## List of changes

I've been contacted by numerous people saying the fact we dropped the support for .NET Framework 4.8 has blocked them from upgrading to latest RC versions
With this PR we should be able to have backward compatability with the Net48 

## Types of changes

What types of changes are you proposing/introducing to the .NET client?
_Put an `x` in the boxes that apply_

- [ ] Bugfix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] New feature (non-breaking change which adds value to the project)
- [ ] Breaking change (fix or feature that would cause existing functionality not to work as expected)

## Documentation
- [ ] Have you proposed a file change/ PR with appium to update documentation? 
## Integration tests
- [ ] Have you provided integration tests to pass against the beta version of appium? (for Bugfix or New feature)

## Details

Please provide more details about changes if it is necessary. If there are new features you can provide code samples showing how they work and possible use cases. Also, you can create [gists](https://gist.github.com) with pasted C# code samples or put them here using markdown. 
About markdown please read [Mastering markdown](https://guides.github.com/features/mastering-markdown/) and [Writing on GitHub](https://docs.github.com/en/get-started/writing-on-github)

### CODE REVIEW

**Comment by Dor-bl** [comment]
@MartyIX your feedback whould be great if possible.

File: test/integration/Appium.Net.Integration.Tests.csproj [hunk 1]
```diff
@@ -1,6 +1,7 @@
-<Project Sdk="Microsoft.NET.Sdk">
+﻿<Project Sdk="Microsoft.NET.Sdk">
   <PropertyGroup>
-    <TargetFrameworks>net6.0</TargetFrameworks>
+    <TargetFrameworks>net48;net6.0</TargetFrameworks>
+	<LangVersion>latest</LangVersion>
```
[the remark this answers is not in the document]
Comment (Dor-bl) [reply test/integration/Appium.Net.Integration.Tests.csproj:4]: No specific reason. 
it's just that I had a compiler error that required to update to Lang version > 8.3, I assume I can align the two projects and use latest on both.

File: src/Appium.Net/Appium/Service/AppiumLocalService.cs [hunk 2]
```diff
@@ -302,11 +318,21 @@ private async Task<bool> PingAsync(TimeSpan span)
...
+#if NET48
+        private async Task<HttpWebResponse> GetHttpResponse(Uri status)
         {
-            HttpResponseMessage response = await SharedHttpClient.GetAsync(status).ConfigureAwait(false);
-            return response;
+            return await Task.Run(() =>
+            {
+                HttpWebRequest request = (HttpWebRequest)WebRequest.Create(status);
+                return (HttpWebResponse)request.GetResponse();
+            });
```
[the remark this answers is not in the document]
Comment (Dor-bl) [reply src/Appium.Net/Appium/Service/AppiumLocalService.cs:328]: @MartyIX, This won't work
`Error	CS1061	'WebRequest' does not contain a definition for 'ConfigureAwait' and no accessible extension method 'ConfigureAwait' accepting a first argument of type 'WebRequest' could be found (are you missing a using directive or an assembly reference?)	`

[the remark this answers is not in the document]
Comment (Dor-bl) [reply src/Appium.Net/Appium/Service/AppiumLocalService.cs:328]: NVM, I can add it here:
```
            return await Task.Run(() =>
            {
                HttpWebRequest request = (HttpWebRequest)WebRequest.Create(status);
                return (HttpWebResponse)request.GetResponse();
            }).ConfigureAwait(false);
```

File: src/Appium.Net/Appium/Service/AppiumLocalService.cs [hunk 3]
```diff
@@ -37,7 +39,9 @@ public class AppiumLocalService : ICommandServer
         private readonly int Port;
         private readonly TimeSpan InitializationTimeout;
         private readonly IDictionary<string, string> EnvironmentForProcess;
+#if !NET48
```
Comment (KazuCocoa) [inline src/Appium.Net/Appium/Service/AppiumLocalService.cs:42]: Do you have any idea when we can drop this...?
I'm not so familiar with .net version env, so it is helpful to leave some comments, or when we could drop this in README.

For example, usually .net48 could be maintained until YYYY MM DD, etc... so we will drop this then, etc..

**Review by KazuCocoa — APPROVED** [review]

**Pull request merged**